lab_1/main.py

```diff
@@ -87,7 +114,17 @@ def get_adjacent_words(tokens: list, word: str, left_n: int, right_n: int) -> li
     right_n = 3
     --> [['man', 'is'], ['dog, 'cat']]
     """
-    pass
```

Comment:
There should be checks for input arguments. Smth like:
`if not isinstance(input_arg_name, expected_type):`
`do something here`